core/chains/evm/logpoller/orm.go

```diff
@@ -129,3 +129,83 @@ func (o *ORM) SelectLogsByBlockRangeFilter(start, end int64, address common.Addr
 	}
 	return logs, nil
 }
+
+// LatestLogEventSigsAddrsWithConfs finds the latest logs by block that matches a list of addresses and list of events
+func (o *ORM) LatestLogEventSigsAddrsWithConfs(fromBlock int64, addresses []common.Address, eventSigs []common.Hash, confs int, qopts ...pg.QOpt) ([]Log, error) {
+	var logs []Log
+
+	var latestBlocks []int64
+
+	arg := map[string]interface{}{
+		"addresses": addresses,
+		"sigs":      eventSigs,
+		"confs":     confs,
+		"chainid":   utils.NewBig(o.chainID),
+		"fromBlock": fromBlock,
+	}
+
+	// Get the latest block with a matching update for each address
+	query, args, err := sqlx.Named(`
+	SELECT MAX(block_number) as block_number FROM logs
+	WHERE evm_chain_id = :chainid
+	AND address IN (:addresses)
```

Comment:
just to clarify this is going to take the latest log for every (address, event_sig) combination right? If so let's make that clear in in the function doc

Reply:
Done.